src/views/open-api-docs-view-v2/types.ts

```diff
@@ -3,23 +3,38 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-// Types
+// Third-party
+import type { NextParsedUrlQuery } from 'next/dist/server/request-meta'
 import type { OpenAPIV3 } from 'openapi-types'
-import type { OperationContentProps } from './components/operation-content'
+// App-wide types
+import type { ApiDocsVersionData } from 'lib/api-docs/types'
+import type { BreadcrumbLink } from '@components/breadcrumb-bar'
+import type { GithubDir } from 'lib/fetch-github-file-tree'
+import type { ProductData, ProductSlug } from 'types/products'
+// Local types
 import type { LandingContentProps } from './components/landing-content'
+import type { VersionSwitcherProps } from '@components/version-switcher'
+import type { OperationContentProps } from './components/operation-content'
 import type { OperationObject } from './utils/get-operation-objects'
-import type { ProductData, ProductSlug } from 'types/products'
-import type { BreadcrumbLink } from '@components/breadcrumb-bar'
 
 /**
  * Shared props are common to both the "landing" and "operation" views.
  */
 export interface SharedProps {
```

Comment:
Updates types to accommodate changes in other files. I've tried to be thorough with type comments - definitely flag if anything's unclear! 🙇‍♂️